sdk/python/feast/client.py

```diff
@@ -165,11 +165,11 @@ def _serving_service(self):
             channel = create_grpc_channel(
                 url=self._config.get(CONFIG_SERVING_URL_KEY),
                 enable_ssl=self._config.getboolean(CONFIG_SERVING_ENABLE_SSL_KEY),
-                enable_auth=False,
+                enable_auth=self._config.getboolean(CONFIG_CORE_ENABLE_AUTH_KEY),
```

Comment:
Shouldnt this be `CONFIG_SERVING_ENABLE_AUTH_KEY`?

Reply:
Should we rename this to `CONFIG_ENABLE_AUTH_KEY` instead of core or serving? 
Since, enabling it just passes metadata and doesn't hurt even if it is disabled on serving and enabled on core. The reason being introduction of more variables.
For ex, since Core and Serving could be set up to have two different auth providers we might have to add variables for that too. I feel at some point we will have to start merging some of the variables which are common. If we want Keep them separate, it would make more sense to separate them as `CoreClient` and `ServingClient`.

Reply:
Ok, lets merge them. That makes sense to me.

Reply:
Yes I will update the release notes.